### ISSUE TITLE
typo in coding-standards.xhtml

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8076]
|<img alt="fdrake's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| fdrake reported|
|-|-|
|Trac ID|trac#440|
|Type|defect|
|Created|2003-12-18 05:23:12Z|

<details><summary>Searchable metadata</summary>

```
trac-id__440 440
type__defect defect
reporter__fdrake fdrake
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__documentation documentation
time__1071724992000000 1071724992000000
changetime__1470421807913474 1470421807913474
version__ 
owner__ 
cc__spiv cc__fdrake
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="fdrake's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|fdrake commented|
|-|-|

```
#!html
<pre>
In doc/howto/coding-standards.xhtml, in the section "Modules", the third
paragraph contains a typo:

"have imports i look like this:"

should probably be:

"have imports which look like this:"
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
Fixed in revision 1.4.  Thanks!
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset d2263cacadbec9d8c3b35b2b01abe03f37734c47
```
#!CommitTicketReference repository="" revision="d2263cacadbec9d8c3b35b2b01abe03f37734c47"
Merge pull request #440 from twisted/8707-rodrigc-getargspec-py3

Author: rodrigc
Reviewer: Lukasa
Fixes: #8707

Do not use inspect.getargspec() in twisted/web/test/test_static.py on Python 3
```
